tests/e2e/cucumber/multiple-interactions.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+let resultsList: string[] = [];
+let newResultsList: string[] = [];
+
+Then('related results are displayed', () => {
+  resultsList = [];
+  cy.getByDataTest('search-grid-result')
+    .should('be.visible')
+    .should('have.length.at.least', 1)
+    .getByDataTest('result-title')
+    .each($resultTitle => {
+      resultsList.push($resultTitle.text());
+    });
+});
+
+Then('new related results are displayed', () => {
+  cy.getByDataTest('search-grid-result')
+    .should('be.visible')
+    .should('have.length.at.least', 1)
+    .getByDataTest('result-title')
+    .each($resultTitle => {
+      newResultsList.push($resultTitle.text());
+    });
+});
+
+Then('new related results are different from previous ones', () => {
+  console.log(resultsList[0]);
+  console.log(newResultsList[0]);
+  expect(newResultsList.every(item => resultsList.includes(item))).to.eq(false);
+  resultsList = Object.assign([], newResultsList);
```

Comment:
do we need this? doing `resultList = newResultList` should be enough.

Also, do we want to keep the console.log ?